### ISSUE TITLE
what is <shellcode-address> ?

### ISSUE THREAD

**Issue by knight2008** [nescio007/teether#5]
`python bin/gen_exploit.py    help 
Usage: bin/gen_exploit.py [flags] <code> <target-address> <shellcode-address> <target_amount> [savefile] [initial-storage file] [initial-balance]`
## issue:
Is target-address the receive address? 
But what is shellcode-address, contract address ? 
Thanks a lot.

**Comment by nescio007** [comment]
Yes, `target-address` is the address that should receive the funds.

`shellcode-address` is the address of a contract that will be executed by exploits using `CALLCODE` or `DELEGATECALL` in the context of the vulnerable contract.
